docker/release/build/Dockerfile-latest

```diff
@@ -1,7 +1,6 @@
 # Use the official Docker Hub Ubuntu 20.04 base image
 FROM ubuntu:20.04
 
-MAINTAINER Timesketch <timesketch-dev@googlegroups.com>
```

Comment:
Not sure why this is removed here

Reply:
The `MAINTAINER` directive has been deprecated in docker-compose, thats why we have removed it.